extensions/wikia/AdEngine/js/template/modalOasisHandler.js

```diff
@@ -22,16 +23,39 @@ define('ext.wikia.adEngine.template.modalOasisHandler', [
 
 		uiFactory.init('modal').then((function (uiModal) {
 			uiModal.createComponent(modalConfig, (function (modal) {
-				var closeBtnDelay = parseInt(closeButtonDelay, 10) * 1000 || 0;
+				var closeBtnDelay = parseInt(closeButtonDelay, 10) * 1000 || 0,
```

Comment:
Ack, sorry. We don't need to multiply by `1000` anymore and I'm thinking of renaming `closeBtnDelay` to `isCloseButtonHidden`. What do you think about it? My thoughts are that it's just confusing that we have `closeBtnDelay` and `closeButtonDelay`.